content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,30 +5,165 @@ weight = 20
 
 +++
 
-## What is Kubeflow?
+This guide describes how to install Kubeflow standalone components or Kubeflow Platform using package
+distributions or raw manifests.
 
-Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
-Operators can choose what is best for their users, there is no requirement to deploy every component.
+Read [the introduction guide](/docs/started/introduction) to understand what are Kubeflow
+standalone components and what is Kubeflow Platform.
 
-Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
-[Architecture](/docs/started/architecture/) pages.
+You can install Kubeflow using one of these methods:
 
-## How to install Kubeflow?
+- [**Install Kubeflow Components Standalone**](#install-kubeflow-components-standalone)
 
-Anywhere you are running Kubernetes, you should be able to run Kubeflow.
-There are two primary ways to install Kubeflow:
+- Install Kubeflow Platform
+  - [**From Packaged Distributions**](#from-packaged-distributions)
+  - [**From Raw Manifests**](#from-raw-manifests) <sup>(advanced users)</sup>
 
-1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
-1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
+## Install Kubeflow Components Standalone
 
-<a id="packaged-distributions"></a>
-<a id="install-a-packaged-kubeflow-distribution"></a>
+Some components in the [Kubeflow ecosystem](/docs/started/architecture/#conceptual-overview) may be
+deployed as standalone services, without the need to install the full platform. You might integrate
+these services as part of your existing AI/ML platform or use them independently.
 
-## Packaged Distributions of Kubeflow
+This is a quick and easier method to get started with Kubeflow ecosystem since those components usually
+don't require additional management tools used in a Kubeflow Platform.
```

Comment:
I've already asked community, distribution, users to comment on this PR if they have concerns with word: easier.
We can wait by end of this week for other objections.

As @jbottum said on the community call it is obvious that starting with single component will be easier than install all Kubeflow Platform.

Reply:
We can't know what is "easier" for any particular user as this is subjective and depends on what they are trying to do.

@rareddy @kimwnasptd can you please clarify what the positions of Red Hat and Canonical are here? (Or any other distributions)

---

The specific wording is not important, and it's much less controversial if we just say standalone components are "a quick way to get started with Kubeflow" (note, this is not "quicker").

@andreyvelich I hope you understand why I am opposed to making making superlative statements, and prefer we stick to factual ones.

Although I doubt we will get many responses here because this PR has SOOO much activity on it, and this comment thread will get lost.